src/webui/api/synccontroller.cpp

```diff
@@ -399,11 +399,16 @@ void SyncController::maindataAction()
 
     data["torrents"] = torrents;
 
-    QVariantList categories;
-    for (auto i = session->categories().cbegin(); i != session->categories().cend(); ++i)
-        categories << i.key();
+    QVariantHash categoriesHash;
+    const auto categories = session->categories();
+    for (auto key : categories.keys()) {
```

Comment:
Please use iterator-based loop (as it was before). It will be faster. Don't forget that this action is called frequently.
```c++
for (auto it = categories.cbegin(); it != categories.cend(); ++it)
```